docs/self-managed/components/orchestration-cluster/identity/debugging-authentication.md

```diff
@@ -0,0 +1,129 @@
+
+2. **Establish Orchestration Cluster user context**
+   - Input: Spring `Authentication`
+   - Output: `CamundaAuthentication` object that carries the user's groups, roles, tenant memberships, etc.
+   - Layer: Orchestration Cluster authentication
+
+3. **Apply authorizations**
+   - Input: `CamundaAuthentication`
+   - Output: Application data, based on applying authorizations according to the user’s memberships
+   - Layer: Orchestration Cluster search layer and workflow engine
+
+Each of these steps may either reject the request or not grant access to the desired data. For example:
+
+- If you use basic authentication and provide an incorrect username/password, the request will fail in step 1.
+- If your user cannot see data because they are not a member of the correct role, the cause lies in step 2.
+- If your user has the right memberships but authorizations are not yet configured, the cause lies in step 3.
+
+You can use the following resources to identify where a request may fail:
+
+- [Reviewing logs](#reviewing-logs)
+- [Reviewing data](#reviewing-data)
+- [Reviewing configuration](#reviewing-configuration)
+
+## Reviewing logs
+
+To obtain debug logs for request authentication and authorization, set logging levels as follows:
+
+<Tabs groupId="optionsType" defaultValue="env" queryString values={[{label: 'Application.yaml', value: 'yaml' }, {label: 'Environment variables', value: 'env' }]}>
+<TabItem value="yaml">
+
+```yaml
+logging.level:
+  org.springframework.security: TRACE
+  io.camunda:
+    authentication: DEBUG
+    security: DEBUG
+```
+
+</TabItem>
+<TabItem value="env">
+
+```
+LOGGING_LEVEL_ORG_SPRINGFRAMEWORK_SECURITY=TRACE
+LOGGING_LEVEL_IO_CAMUNDA_AUTHENTICATION=DEBUG
+LOGGING_LEVEL_IO_CAMUNDA_SECURITY=DEBUG
+```
+
+</TabItem>
+</Tabs>
+
+With these settings, you can trace how requests are handled and how the Spring Security filter chains determine whether a request is authenticated.
+
+## Reviewing data
+
+To review the assignment of users and clients to roles, groups, or tenants—as well as which authorizations are in place—you can use the [Identity UI](/components/identity/identity-introduction.md).
+
+If you do not have access to the API, you can also check the same data in the following Elasticsearch/OpenSearch indexes:
+
+- camunda-authorization
+- camunda-group
+- camunda-mapping-rule
+- camunda-role
+- camunda-tenant
+- camunda-user
+- camunda-web-session
+
+## Reviewing configuration
+
+To review the effective configuration of your Orchestration Cluster, you can call the [Spring Boot Actuator endpoint](https://docs.spring.io/spring-boot/reference/actuator/endpoints.html#actuator.endpoints) at `<server>:<port>/actuator/configprops`.
+
+For example, with a Camunda 8 Run installation, this endpoint is available at `http://localhost:9600/actuator/configprops`. Note that the actuator port differs from the Orchestration Cluster API port and may not always be accessible, depending on your deployment setup.
```

Comment:
❓ Should we also note here that the `/actuator/configprops` should be aimed at the Orchestration cluster with context path if applicable? It may make things less clear though/harder to describe.

I mention this because it has come up recently in an internal [request](https://camunda.slack.com/archives/C08MRKHJ0CD/p1758899257601269)

Reply:
I'm adding a sentence that people may have to replace the localhost:9600 part, but I won't describe in detail what it is for each setup.